main.go

```diff
@@ -348,6 +348,7 @@ func {{.FunctionPrefix}}FSByte(useLocal bool, name string) ([]byte, error) {
 		if err != nil {
 			return nil, err
 		}
+		defer f.Close()
```

Comment:
No need for this to be a defer. Save the results of ReadAll, close, then return.